packages/schema/bind/src/bindings/assemblyscript/wasm/templates/module-type/module-ts.mustache

```diff
@@ -0,0 +1,43 @@
+import * as Types from "..";
+
+{{> serialization_imports}}
+
+import {
+{{#methods}}
+  deserialize{{name}}Args,
+  serialize{{name}}Result{{^last}},{{/last}}
+{{/methods}}
+} from "./serialization";
+
+import {
+  wrap_invoke_args,
+  wrap_invoke_result,
+  wrap_invoke_error,
+  wrap_load_env,
+  InvokeArgs,
+} from "@polywrap/wasm-as";
+
+export abstract class IModule {
```

Comment:
I don't think abstract classes should be prefixed with `I`

Reply:
While I agree, I'm not sure what we should call this. A few options I could think of:
```typescript
export class Module extends BaseModule { }
```
```typescript
export class Module extends ModuleMethods { }
```
```typescript
export class Module extends WrapModule { }
```
```typescript
export class Module extends IModule { }
```
```typescript
export class Module extends Module.Interface { }
```
?